### ISSUE TITLE
Updating volume of each audio track in a video on the fly as well as create a new file with final changes

### ISSUE THREAD

**Issue by Saket7042** [google/ExoPlayer#11332]
I'm working on shorts/reels creation feature in that we have 3 different audio. Which are these

Original video sound
Voiceover which can be provided by user's mic
External audio clip which can be selected by user

I've to modify this on the fly(during playback) as well as create a new video file once user done with final changes. I'll look into this [ChannelMixingAudioProcessor](https://github.com/androidx/media/blob/release/libraries/common/src/main/java/androidx/media3/common/audio/ChannelMixingAudioProcessor.java).

So, to achieve this I've to pass volume float value to each Custom ChannelMixingAudioProcessor class for each audio tracks.

Just guessing there will be a few issues

I've to identify what is the type(Original, Voiceover, External audio) of audio track and based on the type what volume should be set.
As I'm trying to modify it on the fly and playing on Exoplayer so processing time may be long. 

Please suggest some approach to implement this feature. Can someone can post an example by using ChannelMixingAudioProcessor that would be helpful.

**Comment by rohitjoins** [comment]
The com.google.android.exoplayer2 project is now deprecated. We are not accepting any new issues in this project. Please file your issue in the androidx.media3 GitHub project instead: https://github.com/androidx/media/issues
